arb_os/gasAccounting.mini

```diff
@@ -45,7 +45,7 @@ use std::queue::queue_get;
 use std::queue::queue_getOrDie;
 
 use chainParameters::chainParams_millisecondsPerBlock;
```

Comment:
I don't think we use the `chainParams_millisecondsPerBlock` include anymore. With that removed I don't think the method is used anywhere